packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.js

```diff
@@ -54,30 +49,53 @@ const defaultProps = {
 
 class Dropdown extends React.Component {
   render() {
-    const { className, children, isOpen, toggle, direction, onSelect, isPlain, position, id, ...props } = this.props;
+    const {
+      className,
+      children,
+      dropdownItems,
+      isOpen,
+      toggle,
+      direction,
+      onSelect,
+      isPlain,
+      position,
+      ...props
+    } = this.props;
+    const id = toggle.props.id || `pf-toggel-id-${currentId++}`;
```

Comment:
this will be updated to use the unique id wrapper? and nit spelling: toggle on the prefix